tools/convert/main.go

```diff
@@ -256,6 +255,79 @@ func loadRulesMetadata() *config.Metadata {
 	return m
 }
 
+func ConvertConfig(tmplData *configTemplateData, w io.Writer) {
+	tmpl := template.New("configV2.tmpl")
+	tmpl.Funcs(helpers())
+	tmpl, err := tmpl.ParseFS(filesystem, "templates/configV2.tmpl")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "template error %v\n", err)
+		os.Exit(1)
+	}
+
+	err = tmpl.Execute(w, tmplData)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "template error %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func ConvertHelm(tmplData *configTemplateData, w io.Writer) {
+	// convert config if we have it
+	helmConfigAny, ok := tmplData.Data["config"]
+	if ok {
+		helmConfig, ok := helmConfigAny.(map[string]any)
+		if !ok {
+			panic("config in helm chart is the wrong format!")
+		}
+		convertedConfig := &bytes.Buffer{}
+		// make a copy of this tmplData and overwrite the Data part
+		config := *tmplData
+		config.Data = helmConfig
+		// convert the config into the buffer
+		ConvertConfig(&config, convertedConfig)
```

Comment:
The values.yaml `config` section may contain a field named `RulesConfigMapName` that is now incompatible with Refinery's stance against extraneous configuration fields.  We'll need to remove the entry and move it to the root of the values.yaml.

Reply:
Whoops, I completely forgot about that one! Will fix.